frontend/index.html

```diff
@@ -3,83 +3,148 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/orders101-ui.css">
     <title>DIYToolWorld</title>
 </head>
 <body>
     <!-- Account management and contact us section -->
-    <section id="account">
-        <div>
-            <ul>
-                <li><a href="#account">Account</a></li>
+    <header class="main-theme">
+        <div class="account container">
+            <ul class = account-links>
+                <li><a href="#account"><img src="./assets/images/person-fill.svg">Account</a></li>
```

Comment:
In this case I would consider the image to be [decorative](https://www.w3.org/WAI/tutorials/images/decorative/), meaning that it doesn't contain additional information than what is already given by its context. 

For accessibility reasons we should then add an empty `alt` attribute, to communicate the decorative nature of it (versus when the image acts as informative content, and the alternative text contains a description of the image).

Reply:
Additional comment re Trending products: add alt text